quickwit/quickwit-cli/src/index.rs

```diff
@@ -89,7 +89,9 @@ pub fn build_index_command<'a>() -> Command<'a> {
             Command::new("ingest")
                 .about("Indexes JSON documents read from a file or streamed from stdin.")
                 .args(&[
-                    arg!(--index <INDEX> "ID of the target index"),
+                    arg!(--index <INDEX> "ID of the target index")
+                        .display_order(1)
+                        .required(true),
```

Comment:
By default, arg is required so the pattern is more like use `required(false)` to make it optional. The exception I know is when you use `.conflicts_with()`

Reply:
Ok, I had doubts that args is required by default. 

So that means there is nothing to do for this task: `Make required options such as config or index actually required`.

I will remove the commit to get it back as it was.